packages/sw-lib/src/lib/error-factory.js

```diff
@@ -21,6 +21,9 @@ const errors = {
     'string, a Regex to capture request URLs or a Route instance.',
   'empty-express-string': 'The Express style route string must have some ' +
     'characters, an empty string is invalid.',
+  'bad-revisioned-cache-list': `The 'cacheRevisionedAssets()' method expects` +
```

Comment:
This is fine for now, but longer term, we should think about including URLs with links to published documentation about the expected input, rather than inline examples of the type of input we expect. That way if we end up changing the expected format we just need to adjust the definitive JSDocs and don't have to worry about updating error strings.